README.md

```diff
@@ -1,207 +1,249 @@
+It is recommended to define `accelerator` with one of the following values:
 
-[stac-ext-proc]: https://github.com/stac-extensions/processing#suggested-processing-levels
+- `amd64` models compatible with AMD or Intel CPUs (no hardware specific optimizations)
+- `cuda` models compatible with NVIDIA GPUs
+- `xla` models compiled with XLA. models trained on TPUs are typically compiled with XLA.
+- `amd-rocm` models trained on AMD GPUs
+- `intel-ipex-cpu` for models optimized with IPEX for Intel CPUs
+- `intel-ipex-gpu` for models optimized with IPEX for Intel GPUs
+- `macos-arm` for models trained on Apple Silicon
 
-#### Data Type Enum
-
-It is recommended to use the [STAC Raster Extension - Raster Band Object][stac-ext-raster-band-obj]
-in STAC Collections and Items that refer to a STAC Item using `dlm`'s `data_type`. The values should be one of the known
-data types defined by `raster:bands`'s `data_type` as presented in [Data Types][stac-ext-raster-dtype].
+[stac-asset]: https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#asset-object
 
-If source imagery has different `data_type` values than the one specified by `dlm`'s `data_type` property,
-this should provide
-an indication that the source imagery might require a preprocessing step (scaling, normalization, conversion, etc.)
-to adapt the samples to the relevant format expected by the described model.
+### mlm:model Asset
 
-[stac-ext-raster-dtype]: https://github.com/stac-extensions/raster/#data-types
-[stac-ext-raster-band-obj]: https://github.com/stac-extensions/raster/#raster-band-object
+| Field Name | Type     | Description                                                               |
+|------------|----------|---------------------------------------------------------------------------|
+| title      | string   | Description of the model asset.                                           |
+| href       | string   | Url to the checkpoint or model artifact.                                  |
+| type       | string   | "application/x-pytorch" or specify another appropriate custom media type. |
+| roles      | [string] | Specify one or more of ["model", "weights", "compiled"]                   |
 
-#### No Data Value
 
-It is recommended to use the [STAC Raster Extension - Raster Band Object](https://github.com/stac-extensions/raster/#raster-band-object)
-in STAC Collections and Items that refer to a STAC Item using `dlm`'s `nodata`. This value should either map
-to the `raster:bands`'s `nodata` property of relevant bands, or a classification label value representing
-a "*background*" pixel mask (see [STAC Label Extension - Raster Label Notes][stac-ext-raster-label])
-from a `label:type` defined as `raster` with the relevant `raster` asset provided.
+Recommended asset `roles` include `weights` for model weights that need to be loaded by a model definition and `compiled` for models that can be loaded directly without an intermediate model definition.
```

Comment:
Applied in https://github.com/rbavery/dlm-extension/pull/2

Did not include `mlm:compiled: bool` since it seems too specific to certain use cases to be applied at the STAC Item level.
Can still use `mlm:compiled` though in the Asset roles and/or define `mlm:artifact_type: torch.compile`.